pkg/controllers/awsloadbalancercontroller/status.go

```diff
@@ -34,42 +39,25 @@ func (r *AWSLoadBalancerControllerReconciler) updateControllerStatus(ctx context
 	return nil
 }
 
-func credentialRequestsConditions(cr *cco.CredentialsRequest, generation int64) []metav1.Condition {
+func credentialRequestsConditions(cr *cco.CredentialsRequest, secretProvisioned bool, generation int64) []metav1.Condition {
 	var conditions []metav1.Condition
```

Comment:
I don't see the credentials request being used anywhere. Do we still have to pass it to this function.

Reply:
Also the condition name now doesn't make sense. We should call it something like `CredentialsSecretAvailable`.

Reply:
> I don't see the credentials request being used anywhere. Do we still have to pass it to this function.

Changed argument to be just the secret name. 

> Also the condition name now doesn't make sense. We should call it something like CredentialsSecretAvailable.

Fixed